db/migrate/20190719003405_remove_is_suspended_from_users.rb

```diff
@@ -0,0 +1,5 @@
+class RemoveIsSuspendedFromUsers < ActiveRecord::Migration[5.2]
+  def change
+    remove_column :users, :is_suspended, :boolean
```

Comment:
So since none of these migrations have been merged to the master (or staging) branch, you don't need multiple migrations to change the name of this column. You could just change the first migration, delete the schema.rb file, and start over with a clean database. But why the change? `is_suspended` is probably a better name, as it shows right there in the name that it's a boolean. Also, why remove the default value?

Reply:
We tried to follow the guidelines from the file you sent us. We had not idea if the default value for is_suspended would affect the results. Since we rake db:migrate, what would be the best commands to get back to is_suspended?